client/hosting/performance/components/PerformanceSubtitleWithLink.tsx

```diff
@@ -0,0 +1,26 @@
+import { useTranslate } from 'i18n-calypso';
+import InlineSupportLink from 'calypso/components/inline-support-link';
+import isA8CForAgencies from 'calypso/lib/a8c-for-agencies/is-a8c-for-agencies';
+
+export const PerformanceSubtitleWithLink = () => {
+	const translate = useTranslate();
+
+	const link = isA8CForAgencies() ? (
+		<a
+			href="https://agencieshelp.automattic.com/knowledge-base/check-your-sites-performance"
+			target="_blank"
+			rel="noreferrer"
+		/>
+	) : (
+		<InlineSupportLink supportContext="site-performance" showIcon={ false } />
```

Comment:
I added this as a question on Slack, but can't we reuse the `InlineSupportLink` and simply replace the link for our use-case?

Reply:
I would say here to add the support link to `client/components/inline-support-link/context-links.js` following the examples there and then in the `InlineSupportLink` `supportContext` to either "site-performance" or the a4a specific once added to the `context-links.js` file and remove this new component.

Reply:
Thanks for the feedback! We don't use `InlineSupportLink` on A4A yet, and the articles in the main section are all related to WordPress.com. This will cause a lot of confusion for the agencies. 

On a side note, we might have to build a similar component for A4A, but this might happen later.

Reply:
Thanks, @vitozev! I have updated the PR